dynawo/3rdParty/CMakeLists.txt

```diff
@@ -8,9 +8,6 @@
 #
 # This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
 
-# Minimum required (for ExternalProject_Add)
-cmake_minimum_required(VERSION 3.9.6 FATAL_ERROR)
```

Comment:
You say there is a top cmake_minimum_requirement but I see it is deleted here

Reply:
Absolutely right, it is my mistake !

Reply:
I have a new problem too, that I don't know why travis failed. DO you have an idea ?

Reply:
The error is the following:
[ 29%] Performing configure step for 'libxml2'

You must have libtool installed to compile libxml.

Download the appropriate package for your distribution,

or see http://www.gnu.org/software/libtool

make[2]: *** [CMakeFiles/libxml2.dir/build.make:108: stamp_dir/libxml2-configure] Error 1

Reply:
thanks for the digging in the log. it means we need to add a new requirement in the linux libraries. So I also need to update the docker for travis

Reply:
I updated the Dockerfile so tomorrow we will be able to launch the tests again

Reply:
Thanks too. libtool is already part of linux requirements in dynawo (see Building Dynaωo from sources on Linux from dynawo.org)

Reply:
I reran the build and still the same error.

Reply:
I will have a look at it